l1-contracts/src.ts/deploy-test-process.ts

```diff
@@ -104,10 +105,25 @@ export async function initialTestnetDeploymentProcess(
   // deploy the verifier first
   await initialBridgehubDeployment(deployer, extraFacets, gasPrice, true);
   await initialBridgehubDeployment(deployer, extraFacets, gasPrice, false);
-  await registerHyperchain(deployer, false, extraFacets, gasPrice, baseTokenName);
+  await registerHyperchainWithBridgeRegistration(deployer, false, extraFacets, gasPrice, baseTokenName);
   return deployer;
 }
 
+export async function registerHyperchainWithBridgeRegistration(
+  deployer: Deployer,
+  onlyVerifier: boolean,
+  extraFacets: FacetCut[],
+  gasPrice: BigNumberish,
+  baseTokenName?: string,
+  chainId?: string
+) {
+  chainId = chainId ?? deployer.chainId.toString();
+  await registerHyperchain(deployer, onlyVerifier, extraFacets, gasPrice, baseTokenName, chainId, true);
+  const l1SharedBridge = deployer.defaultSharedBridge(deployer.deployWallet);
+  const upgradeCall = l1SharedBridge.interface.encodeFunctionData("initializeChainGovernance", [chainId, ADDRESS_ONE]);
```

Comment:
why do we set a dummy value here?

Reply:
It is a L1 test process, there is no L2 contract deployed

Reply:
okay